cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
+        &self,
+        data_dir: &PathBuf,
+    ) -> Fallible<()> {
+        let supported_version = ["1.0.0"];
+        let path = data_dir.join("version");
+        let version = tokio::fs::read(&path)
+            .await
+            .context(format!("Reading {:?}", &path))?;
+        let trimmed_version = version.trim();
+
+        for supported_version in supported_versions.iter() {
+            if supported_version == trimmed_version {
+                return Ok(trimmed_version);
+            }
+        }
+
+        return Fallible<trimmed_version,format_err!("unrecognized graph-data version {}; supported versions: {}", trimmed_version, supported_versions.join(", "))>;
```

Comment:
```suggestion
        return Err(format_err!(
            "unrecognized graph-data version {}; supported versions: {:?}",
            trimmed_version,
            supported_versions
        ));
```
`Fallible` is a shortcut to [`Result`](https://doc.rust-lang.org/std/result/), which should be either `Ok` or `Err`. Also Rust can format a vector via `{:?}` without extra `join`

Reply:
Fixed with 7179eefe -> 1b34d2cf.